code/modules/power/lighting.dm

```diff
@@ -419,10 +419,14 @@ var/global/list/obj/machinery/light/alllights = list()
 		update(0)
 
 /obj/machinery/light/attack_ghost(mob/user)
+	return
+/*
+/obj/machinery/light/attack_ghost(mob/user)
 	if(blessed) return
 	src.add_hiddenprint(user)
```

Comment:
Kill this meme until at least all the collaborators can agree on something
Comic just scolded someone for removing `src` in lines he didn't otherwise touch